contracts/HybridCustody.cdc

```diff
@@ -427,13 +469,16 @@ pub contract HybridCustody {
     marking the account as owned by no one. Note that even if there isn't an owner, the parent accounts would still exist, allowing
     a form of Hybrid Custody which has no true owner over an account, but shared partial ownership.
     */
-    pub resource ChildAccount: Account, BorrowableAccount, ChildAccountPublic, ChildAccountPrivate {
+    pub resource ChildAccount: Account, BorrowableAccount, ChildAccountPublic, ChildAccountPrivate, MetadataViews.Resolver {
         priv var acct: Capability<&AuthAccount>
 
         pub let parents: {Address: Bool}
         pub var acctOwner: Address?
         pub var relinquishedOwnership: Bool
 
+        access(self) let data: {String: AnyStruct}
```

Comment:
Mentioned in another comment, but could we index on Type here? Seems like that would make `getViews()` and `resolveView()` easier to manage

Reply:
If we do that, we can't add multiple fields of the same type which could well happen. Imagine I want to add a number, for instance. Keying by type means I can't insert another

Reply:
Added a dictionary for metadata views explicitly which is keyed by Type. I'm afraid that if we combine them, someone might override data and break a metadata view completely